CHANGES.md

```diff
@@ -144,7 +164,7 @@ The .NET Standard 2.0 version of NUnit continues to gain more functionality that
 is found in the .NET 4.5 version of the framework like setting the ApartmentState
 and enabling Timeout on tests.
 
-#### Issues Resolved
+### Issues Resolved
```

Comment:
Same comment as in the other PR - see https://github.com/nunit/nunit/pull/3840#discussion_r621867264